### ISSUE TITLE
NFTs page

### ISSUE THREAD

**Issue by toniengelhardt** [toniengelhardt/merklin#53]

**Comment by RyanPaulGannon** [comment]
Maybe this could be of use? I'm going to have a deeper look at it tomorrow. I saw it on Twitter the other day.

https://www.alchemy.com/sdk

**Comment by RyanPaulGannon** [comment]
@toniengelhardt could you please assign me to this?

**Comment by toniengelhardt** [comment]
Yes, definitely, I'll assign you. Looks very interesting! We just need to make sure that we do not hit any request limits in the free tiers of APIs/SDKs for now.

**Comment by RyanPaulGannon** [comment]
I'm conscious I've been quiet, I've got the NFT data to display as an object. Just working on displaying them and making it pretty. Then tidying up the code, also looking at other ways other than just the Alchemy SDK because as you say, conscious of request limits.

**Comment by toniengelhardt** [comment]
> I'm conscious I've been quiet, I've got the NFT data to display as an object. Just working on displaying them and making it pretty. Then tidying up the code, also looking at other ways other than just the Alchemy SDK because as you say, conscious of request limits.

Amazing! I improved the UI a bit for the charts, they all have options now.

**Comment by toniengelhardt** [comment]
No worries, I'm also super busy atm and haven't had time to work on it, but will hopefully be back at it soon. I've also set up a small landing page ([merklin.xyz](https://merklin.xyz)), not sure if you've seen it.

**Comment by toniengelhardt** [comment]
And congrats on the new contract!

**Comment by RyanPaulGannon** [comment]
Thank you my friend!

I hadn't seen it, really cool man, like it a lot!

**Comment by toniengelhardt** [comment]
I think it's nice to have a little entry point, like if you want to send the project to someone. Will add more info over time.
